sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -26,6 +26,10 @@
 
   <packaging>jar</packaging>
 
+  <properties>
+    <beam.version>0.4.0-incubating-SNAPSHOT</beam.version>
```

Comment:
Is there a way to get this number from the POM configuration? Alternatively, can we make it so this gets updated by the other scripts? Eg, this is a template for the generated pom -- I think we may be able to pick some of these parameters from the parent pom (eg., make the beam.version match the parent version, etc.)